edward/criticisms.py

```diff
@@ -28,6 +28,7 @@ def evaluate(metrics, model, variational, data, y_true=None):
         according to the Stan program's data block.
     y_true : np.ndarray or tf.Tensor
         True values to compare to in supervised learning tasks.
+    n_samples : number of posterior samples
```

Comment:
The first line in the doc string would be the type, e.g., `n_samples : int, optional` followed by `Number of posterior samples.`